src/components/task-step/all-steps.cjsx

```diff
@@ -109,29 +109,29 @@ ExternalUrl = React.createClass
   displayName: 'ExternalUrl'
   mixins: [StepMixin, StepFooterMixin]
   hideContinueButton: -> true
+  getDefaultProps: ->
+    redirectToUrl: (url) -> window.open( url, '_blank')
```

Comment:
why open this using `window.open` instead of just making a link with `target='_blank'`?